rxmongo/src/main/scala/akka/contrib/persistence/mongodb/RxMongoPersistenceExtension.scala

```diff
@@ -163,6 +163,16 @@ class RxMongoDriver(system: ActorSystem, config: Config) extends MongoPersistenc
       name = Some(indexName)))
     collection
   }
+
+  override private[mongodb] def cappedCollection(name: String)(implicit ec: ExecutionContext) = {
+    val collection = db[BSONCollection](name)
+    collection.stats().flatMap{ case stats if ! stats.capped =>
```

Comment:
Now that I go back through, this should come from the config, right?  `realtime_collection_size`?